### ISSUE TITLE
How do I change to a regular single boot setup?

### ISSUE THREAD

**Issue by giddyhup** [maxnet/berryboot#182]
I would like to change to a classical boot setup, how can this be accomplished?

**Comment by maxnet** [comment]
There is no easy way to do that.
But if you really want to, you can find some inspiration in this issue: https://github.com/maxnet/berryboot/issues/130

**Comment by giddyhup** [comment]
Thanks for the quick response. I manually mount the target for the backup (a micro sd card in a card reader, unsquashing will be done later in a VM running Debian) but the backup (altered image + data) isn't writing to the selected target, nothing happens.

**Comment by maxnet** [comment]
Try a normal USB stick.

**Comment by giddyhup** [comment]
Thanks again. It would write to a USB stick but complains that it is too small. I thought that 26G should be enough:

```
$ df -h
df: `/squashfs': No such file or directory
Filesystem      Size  Used Avail Use% Mounted on
rootfs           15G  5.6G  8.3G  41% /
dev             407M     0  407M   0% /dev
/dev/mmcblk0p2   15G  5.6G  8.3G  41% /mnt
none             15G  5.6G  8.3G  41% /
tmpfs            84M  392K   83M   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           167M     0  167M   0% /run/shm
/dev/sda1        58G   26G   29G  48% /mnt/USBflash
/dev/mmcblk0p1  127M   37M   91M  29% /media/089E-E3A8
```

(I have my music collection on that stick.)

**Comment by maxnet** [comment]
Is the error "Capacity too small. Need %1 MB" or something else?
If so, how much space does it say you need?

**Comment by giddyhup** [comment]
There error (on a Pi 2) is

```
FATAL ERROR: Mksquashfs requires more physical memory than is available!
```

Would an alternative be to do it offline in a VM or would rsyncing the OS partition after boot be an option?

**Comment by giddyhup** [comment]
```
cp -pdRx
```

worked.

I set up a blank SD card with proper partitions using gparted. I copied the boot files from another Pi 2 to the boot partition, I booted my berryboot setup with the new SD card in a card reader in one of the USB ports. Using the command above I copied the running OS partition to the blank one.

I only had to edit /etc/network/interfaces and enter proper network settings. Additionally I ran into some other minor problems but those stemmed from having run rpi-update and thus having created an inconsistent setup.
